src/modules/about-us/BelieveInPower/ContentSection.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { QuoteIcon } from "@radix-ui/react-icons";
+const ContentSection=()=>{
```

Comment:
1 space would be good here after import.

Reply:
Done.